contract/test/ui-kit-goals/batchQuery.js

```diff
@@ -0,0 +1,176 @@
+// @ts-check
+import { E } from '@endo/far';
+
+/** @typedef {'children' | 'data'} AgoricChainStoragePathKind */
+/** @template T @typedef {import('@endo/marshal').FromCapData<T>} FromCapData<T> */
+/** @template T @typedef {import('@endo/eventual-send').ERef<T>} ERef<T> */
+
```

Comment:
Please add a file comment. Is this a mock implementation of vstorage or something else?

Reply:
moved to #16